zerver/openapi/zulip.yaml

```diff
@@ -22518,6 +22518,9 @@ paths:
         Update the members of a [user group](/help/user-groups). The
         user IDs must correspond to non-deactivated users.
 
+        **Changes**: Prior to Zulip 11.0 (feature level ZF-4479c6), members and subgroups
```

Comment:
Subgroups are added/removed using a different API endpoint.

Reply:
Got it, thanks, changed!